.github/workflows/deploy.yml

```diff
@@ -0,0 +1,52 @@
+name: Build
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  build:
```

Comment:
Let's also add a conditional statement that checks for a `DEPLOY_WITH_DOCKER` (see this [example](https://github.com/sparkbox/sparkpress-wordpress-starter/blob/ae4fb6d24bc6eec1960e122fda661a389def5578/.github/workflows/deploy.pantheon.yml#L10)).

We can set that variable in the [action secrets](https://github.com/sparkbox/sparkpress-wordpress-starter/settings/secrets/actions)